src/main/java/com/ays/institution/controller/InstitutionController.java

```diff
@@ -0,0 +1,47 @@
+package com.ays.institution.controller;
+
+import com.ays.common.model.dto.response.AysResponse;
+import com.ays.institution.model.Institution;
+import com.ays.institution.model.dto.response.InstitutionResponse;
+import com.ays.institution.model.mapper.InstitutionToInstitutionResponseMapper;
+import com.ays.institution.service.InstitutionService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+/**
+ * REST controller class for managing institution-related operations via HTTP requests.
+ * This controller handles the business operations for institutions in the system.
+ * The mapping path for this controller is "/api/v1/institutions".
+ */
+@RestController
+@RequestMapping("/api/v1")
+@RequiredArgsConstructor
+@Validated
+class InstitutionController {
+
+    private final InstitutionService institutionService;
+
+    private final InstitutionToInstitutionResponseMapper institutionToInstitutionResponseMapper = InstitutionToInstitutionResponseMapper.initialize();
+
+    /**
+     * Retrieves a summary of all institutions.
+     * Requires the user to have the 'SUPER_ADMIN' authority.
+     *
+     * @return An {@link AysResponse} containing a list of {@link InstitutionResponse} representing the summary of institutions.
+     */
+    @GetMapping("/institutions/summary")
+    @PreAuthorize("hasAnyAuthority('SUPER_ADMIN')")
+    public AysResponse<List<InstitutionResponse>> getInstitutionsSummary() {
+
+        final List<Institution> institutionsSummary = institutionService.getInstitutionsSummary();
```

Comment:
`institutionsSummary` yerine aktif olan kurumları döndüğümüz için `activeInstitutions` olarak isimlendirmemiz daha iyi olabilir

Reply:
Bu kısmı revise ettim

Reply:
@Rapter1990 bu kısım düzeltilmemiş gibi görünüyor, tekrardan bakabilir misin?

Reply:
Selamlar, düzelttim.